site/layouts/partials/sidebar-tree.html

```diff
@@ -85,7 +85,7 @@
           <!-- always show pages in the current section -->
           {{ if $activeSection }}
               {{ $showPage = true }}
-              {{ $activePage := eq . $p }}
+              {{ $activePage = eq . $p }}
```

Comment:
Using `:=` shadows the outer scope `activePage` variable. So the outer scope variable will always be `false` before this fix.